IntegrationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -1,3 +1,5 @@
+#if 0
```

Comment:
why do we need to explicitly add the module map?

Reply:
This is the workaround for https://bugs.swift.org/browse/SR-15154, if our users encounter the bug, they'll need to include this file in their bridging header.